server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -275,6 +278,41 @@ public static URLClassLoader getClassLoaderForExtension(File extension) throws M
     return loader;
   }
 
+  public static List<URL> getURLsForClasspath(String cp)
+  {
+    try {
+      String[] paths = cp.split(File.pathSeparator);
+
+      List<URL> urls = new ArrayList<>();
+      for (int i = 0; i < paths.length; i++) {
+        File f = new File(paths[i]);
+        if ("*".equals(f.getName())) {
```

Comment:
if you mean using something from jvm to convert provided classpath string to List of URL, then I don't think there is any standard method in the jdk to do that. Please let me know if there is any and I'll remove this.

btw it is intentional that user will be able to configure different classpath for druid and for the hadoop containers.